server-side/.env

```diff
@@ -0,0 +1,5 @@
+PORT=5000
+password=EuACQK5n92JI3vn4
+DB_URL="mongodb://127.0.0.1:27017/TimeOut"
```

Comment:
db url shouldn't be your local route

Reply:
notice: you shouldn't commit changes that you did for yourself